recipes/tiptop/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "tiptop" %}
+{% set version = "0.0.16" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tiptop-{{ version }}.tar.gz
+  sha256: cebdce70347eff7dc621dc0c9bd2167254db75ce490db7ea216cc6d970d858bc
+
+build:
+  # Cannot be noarch due to binary executables being build
```

Comment:
Why is then no compiler listed here?

Reply:
@xhochy  My mistake, I forget the entry point. I can switch back to noarch. It doesn't work with win though.

Reply:
Ah, by "binary executables" you mean entry points? In that case, yes, it should be marked as `noarch` please.

Reply:
It's already been changed to noarch @jaimergp @xhochy @conda-forge/staged-recipes